launcher/ui/pages/instance/ServersPage.cpp

```diff
@@ -88,6 +90,15 @@ struct Server {
         }
     }
 
+    std::tuple<QString, int> splitAddress() const {
+        auto parts = m_address.split(":");
+        if (parts.size() == 1) {
+            return std::make_tuple(parts[0], 25565);
+        } else {
+            return std::make_tuple(parts[0], parts[1].toInt());
+        }
+    }
```

Comment:
I think this is a duplication of the logic here:
https://github.com/PrismLauncher/PrismLauncher/blob/develop/launcher/minecraft/launch/MinecraftTarget.cpp#L21-L65